apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/modules/components/Claimable.tsx

```diff
@@ -57,20 +64,10 @@ export type ClaimConditionValue = {
   auxData: string;
 };
 
-type ClaimCondition = {
-  availableSupply: bigint;
-  allowlistMerkleRoot: `0x${string}`;
-  pricePerUnit: bigint;
-  currency: string;
-  maxMintPerWallet: bigint;
-  startTimestamp: number;
-  endTimestamp: number;
-  auxData: string;
-};
-
 function ClaimableModule(props: ModuleInstanceProps) {
   const { contract, ownerAccount } = props;
   const account = useActiveAccount();
+  const [tokenId, setTokenId] = useState<string>("");
 
   const isErc721 = props.contractInfo.name === "ClaimableERC721";
```

Comment:
Price per token is either not showing the correct data or it is not getting updated 

this is what I submited: 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/aa8f2da2-2954-4b59-8e1e-8324e3e03ed4.png)

this is what I get after page refetch: 
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/2c0e5735-5fbb-4fda-82c9-522e0bbba0f0.png)

Reply:
Addressed with the following code:

```TS
      // default case is zero state, so 0 // 10 ** 18 still results in 0
      pricePerToken: Number(
        toTokens(claimCondition?.pricePerUnit, props.tokenDecimals || 18),
      ),
```

This covers the default case (as mentioned in the comments) since the `pricePerUnit` will always be 0, dividing it by anything will result in zero